src/pkg/reg/model/resource.go

```diff
@@ -33,6 +38,9 @@ type Resource struct {
 	IsDeleteTag bool `json:"is_delete_tag"`
 	// indicate whether the resource can be overridden
 	Override bool `json:"override"`
+	// Skip is a flag for resource which satisfies replication rules but should
+	// be skipped because of other limits like when dest project's type is proxy cache.
+	Skip bool `json:"-"`
```

Comment:
it's better to add an Skip() method since we may add more attribute to control the project filter, no just for proxy cached project.

Reply:
if in struct have a filed `Skip`, we can not add method name same with field.